sql/src/test/java/org/apache/druid/sql/calcite/CalciteWindowQueryTest.java

```diff
@@ -231,6 +233,25 @@ public void windowQueryTestWithCustomContextMaxSubqueryBytes(String filename) th
     }
   }
 
+  @Test
+  public void testWindow()
+  {
+    testBuilder()
+            .sql("SELECT\n" +
```

Comment:
A test case in MSQ window functions would also be helpful.

Reply:
great idea :)
after the separation of test suppliers [patch](https://github.com/apache/druid/pull/16275) gets in that we'll be even easier to do!